public/caesar-shift/decrypt.py

```diff
@@ -0,0 +1,32 @@
+import string
+from encrypt_library import *
+from encrypt import caesar_shift
+#START#
+def caesar_shift_decrypt(ciphertext, shift_value=5):
+  #we can flip the shift value we think the encryptor used to decrypt the message!
+  shift_value = -shift_value
```

Comment:
I see the pedagogical reasons of having this be an explicit step, but I agree it still gets the message across.